src/main/java/org/opensrp/domain/postgres/PgStockAndProductDetails.java

```diff
@@ -7,13 +7,10 @@
 
 @Getter
 @Setter
-@NoArgsConstructor
 @AllArgsConstructor
-public class StockObjectMetadata {
-
-	private Long date_created;
-	private String to_from;
-	private Long date_updated;
-	private long version;
+@NoArgsConstructor
+public class PgStockAndProductDetails {
```

Comment:
This class is duplicate of `org.opensrp.domain.StockAndProductDetails` on path evaluator and should be deleted

Reply:
No, the entities of Stock and Product Catalogue in this domain is of Postgres while on Plan Evaluator stock and product catalogue are domain entities. @githengi 
Since the query would result in the Postgres entity, we cannot utilize the PlanEvaluator's class here.

Reply:
the classes have the same signature and since the class is not auto-generated just use the class on PlanEvaluator's on the mapper and drop this